metagraph/src/common/vectors/wavelet_tree.cpp

```diff
@@ -392,7 +393,10 @@ bool wavelet_tree_sdsl<t_wt_sdsl>::load(std::istream &in) {
 template <class t_wt_sdsl>
 sdsl::int_vector<> wavelet_tree_sdsl<t_wt_sdsl>::to_vector() const {
     sdsl::int_vector<> vector(wwt_.size(), 0, logsigma_);
-    std::copy(wwt_.begin(), wwt_.end(), vector.begin());
+    #pragma omp parallel for num_threads(get_num_threads()) schedule(static, 1024)
```

Comment:
maybe add a comment here that the block size (1024) has to be a multiple of 64, otherwise this loop will lead to race conditions